c++/src/kj/refcount.c++

```diff
@@ -48,7 +48,12 @@ void Refcounted::disposeImpl(void* pointer) const {
 // Atomic (thread-safe) refcounting
 
 AtomicRefcounted::~AtomicRefcounted() noexcept(false) {
+#if defined(__clang__) || !defined(_MSC_VER)
+  KJ_ASSERT(__atomic_load_n(&refcount, __ATOMIC_RELAXED) == 0,
```

Comment:
Similarly here... in correct operation, this thread should have been the one which decremented the refcount to zero. If this thread was the last one to access the value, and no other thread will access it in the future, then it should not be a problem for it to read the value non-atomically. Why is TSAN alerting on this?